src/Credentials.ts

```diff
@@ -528,22 +529,17 @@ class Credentials {
    * @param    {Object[]}    [opts.vc]           An array of JWTs about the requester, signed by 3rd parties
    * @returns  {Promise<Object, Error>}          A promise which resolves with a signed JSON Web Token or rejects with an error
    */
-  createVerificationSignatureRequest(
-    unsignedClaim: object,
-    { aud, sub, riss, callbackUrl, vc, expiresIn }: VerificationRequest
-  ) {
-    return this.signJWT(
-      {
-        unsignedClaim,
-        sub,
-        riss,
-        aud,
-        vc,
-        callback: callbackUrl,
-        type: Types.VERIFICATION_SIGNATURE_REQUEST
-      },
-      expiresIn
-    )
+  createVerificationSignatureRequest(unsignedClaim: object, { aud, sub, riss, callbackUrl, vc, expiresIn}:VerificationRequest) {
+    return this.signJWT({
+      unsignedClaim,
+      sub,
+      riss,
+      aud,
+      vc,
+      callback: callbackUrl,
+      type: Types.VERIFICATION_SIGNATURE_REQUEST,
+      rexp: expiresIn ? expiresIn : undefined,
```

Comment:
So we still need expiresIn to go to `exp`. Also `expiresIn` is `exp: now + expiresIn`.

I would just add `rexp` as a new option and leave `expiresIn` as is.

Reply:
fixed